TicTacToe/TicTacToe.cs

```diff
@@ -32,46 +37,114 @@ public static void GetInput()
             int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
             int column = int.Parse(Console.ReadLine());
+            PlaceMark(row, column);
+            CheckForWin();
+            CheckForTie();
+            playerTurn = (playerTurn == "X") ? "O" : "X";
+
         }
 
         public static void PlaceMark(int row, int column)
         {
-        // your code goes here
+
+            //string[] myRow = board[row];
+            //string myColumn = myRow[column];
+            //myRow[column] = "X";
+            board[row][column] = playerTurn;
+
+
+
+
+
+
+            // your code goes here
         }
 
         public static bool CheckForWin()
-        {
-            // your code goes here
+        { // your code goes here
 
+            if (HorizontalWin() || VerticalWin() || DiagonalWin() == true)
+            {
+                Console.WriteLine("Player " + playerTurn + " Won!");
+                Environment.Exit(0);
+            }
             return false;
+
+
         }
 
         public static bool CheckForTie()
```

Comment:
This method name says to me, "you can call me to determine if the game has ended in a tie". But what it is actually doing is checking if any spots remain unplayed. There is a distinctions there that could lead to trouble for someone else who does not know you wrote this method with the _assumption_ it would always be called _after_ checking for no wins.